prow/plugins/golint/golint_test.go

```diff
@@ -217,6 +217,103 @@ func TestLint(t *testing.T) {
 	}
 }
 
+func TestLintCodeSuggestion(t *testing.T) {
+	var pullFilesForLintNamesUnderscore = map[string][]byte{
+		"qux.go": []byte(`// Package bar comment
+package bar
+
+// Qux_1 comment
+func Qux_1() error {
+	return nil
+}
+`),
+	}
+
+	var pullFilesForLintNamesAllCaps = map[string][]byte{
+		"qux.go": []byte(`// Package bar comment
+package bar
+
+// QUX_FUNC comment
+func QUX_FUNC() error {
+	return nil
+}
+`),
+	}
+
+	var codeChangeForLintNamesUnderscore = "@@ -0,0 +1,7 @@\n+// Package bar comment\n+package bar\n+\n+// Qux_1 comment\n+func Qux_1_Func() error {\n+   return nil\n+}"
+
+	var lintNamesUnderscoreSuggestion = "```suggestion\nfunc Qux1() error {\n```\nGolint naming: don't use underscores in Go names; func Qux_1 should be Qux1. [More info](http://golang.org/doc/effective_go.html#mixed-caps). <!-- golint -->"
+
+	var codeChangeForLintNamesAllCaps = "@@ -0,0 +1,7 @@\n+// Package bar comment\n+package bar\n+\n+// QUX_FUNC comment\n+func QUX_FUNC() error {\n+   return nil\n+}"
+
+	var lintNamesAllCapsSuggestion = "```suggestion\nfunc QuxFunc() error {\n```\nGolint naming: don't use ALL_CAPS in Go names; use CamelCase. [More info](https://golang.org/wiki/CodeReviewComments#mixed-caps). <!-- golint -->"
+
+	var testcases = []struct {
+		codeChange string
+		pullFiles  map[string][]byte
+		suggestion string
+	}{
+		{
+			codeChangeForLintNamesUnderscore,
+			pullFilesForLintNamesUnderscore,
+			lintNamesUnderscoreSuggestion,
```

Comment:
These variables are unnecessary. The values are only used in one place (here) so they just make the code harder to read by adding more indirection. Please just use the literal values here (same for the other test case).